docs/installation-deb.md

```diff
@@ -115,7 +117,7 @@ The log - should you need it - is at /var/log/cassandra/system.log
 
 ## Set up statsd
 
-You can optionally statsd or a statsd-compatible agent for instrumentation of graphite-api.
+You can optionally statsd or a statsd-compatible agent for instrumentation of your applications.
```

Comment:
too vague. maybe make it "of graphite and optionally any of your other applications"